src/scripts/main.js

```diff
@@ -1 +1,73 @@
 'use strict';
+
+const WAIT = 3000;
+const body = document.body;
+
+function createElemPromise(result) {
+  const elem = document.createElement('div');
+
+  elem.setAttribute('data-qa', 'notification');
+
+  if (result) {
+    elem.className = result;
+  }
+
+  return elem;
+}
+
+const elemFirstPromise = createElemPromise();
```

Comment:
The `createElemPromise` function is called without arguments here, which means `result` will be `undefined`. This will result in the created element having no class name initially. Ensure this is the intended behavior.